app/vmalert/manager_test.go

```diff
@@ -23,12 +22,8 @@ func TestManagerUpdateError(t *testing.T) {
 	m := &manager{groups: make(map[uint64]*Group)}
```

Comment:
Please rename test to "TestManagerEmptyGroups" or similar to reflect the purpose of this test.